windows/win_updates.ps1

```diff
@@ -331,13 +331,13 @@ Function RunAsScheduledJob {
 
       # FUTURE: configurable timeout so we don't block forever?
       # FUTURE: add a global WaitHandle in case another instance kills our job, so we don't block forever
-      $job = Wait-Job -Name $schedjob.Name -ErrorAction SilentlyContinue 
+      $job = Wait-Job -Name $schedjob.Name -ErrorAction SilentlyContinue
   }
 
   $sw = [System.Diagnostics.Stopwatch]::StartNew()
 
   # NB: output from scheduled jobs is delayed after completion (including the sub-objects after the primary Output object is available)
-  While (($job.Output -eq $null -or -not ($job.Output | Get-Member -Name Keys) -or -not $job.Output.Keys.Contains('job_output')) -and $sw.ElapsedMilliseconds -lt 15000) {
+  While (($job.Output -eq $null -or -not (Get-Member -InputObjet $job.Output -Name Keys) -or -not $job.Output.Keys.Contains('job_output')) -and $sw.ElapsedMilliseconds -lt 15000) {
```

Comment:
Ah, yes, that is a typo.  Looks like I 'backspaced' after I committed and and confirmed it working on my local repository.

-
> On Mar 7, 2016, at 11:19 AM, Kevin Kenny <notifications@github.com> wrote:
> 
> In windows/win_updates.ps1 <https://github.com/ansible/ansible-modules-extras/pull/1804#discussion_r55237043>:
> 
> >    }
> >  
> >    $sw = [System.Diagnostics.Stopwatch]::StartNew()
> >  
> >    # NB: output from scheduled jobs is delayed after completion (including the sub-objects after the primary Output object is available)
> > -  While (($job.Output -eq $null -or -not ($job.Output | Get-Member -Name Keys) -or -not $job.Output.Keys.Contains('job_output')) -and $sw.ElapsedMilliseconds -lt 15000) {
> > +  While (($job.Output -eq $null -or -not (Get-Member -InputObjet $job.Output -Name Keys) -or -not $job.Output.Keys.Contains('job_output')) -and $sw.ElapsedMilliseconds -lt 15000) {
> Surely InputObjet is a typo? Should be InputObject ?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/ansible/ansible-modules-extras/pull/1804/files#r55237043>.
>